wallet/wallet-cli-lib/src/commands/mod.rs

```diff
@@ -269,6 +269,32 @@ pub enum WalletCommand {
         label: Option<String>,
     },
 
+    #[clap(name = "standalone-add-multisig")]
+    AddStandaloneMultisig {
+        /// The minimum required signatures out of the specified public keys
+        min_required_signatures: u8,
+
+        // Public keys from which to create the multisig challenge
+        public_keys: Vec<String>,
+
+        /// Optionally specify a label to the new address
+        #[arg(long = "label")]
+        label: Option<String>,
+    },
+
+    #[clap(name = "standalone-multisig-utxos")]
+    ListMultisigUtxo {
```

Comment:
We also need a function to list available standalone addresses with their lables, and any relevant information.